school_library_decorator/student.rb

```diff
@@ -1,12 +1,15 @@
-require './person.rb'
-
+require './person'
+# rubocop:disable Style/OptionalBooleanParameter
```

Comment:
- [ ] Please don't disable linter rules. Disabling rubocop might help with linter errors but these are here to help us write clean code.